File: 01_Language/05_Python/tests/thread_/test_thread2.py
# coding: utf-8
import threading
from time import sleep


def my_thread(word):
    sleep(1)
    print(threading.current_thread().getName(), word)


for i in range(5):
    threading.Thread(target=my_thread, args=("say hello",)).start()

print("main thread over")

File: 01_Language/03_Java/j2se/src/test/java/thread/TestThread2.java
package thread;

public class TestThread2 {
    public static void main(String[] args) {
        Ticket t = new Ticket();

        for (int i = 0; i < 10; i++) {
            new Thread(t, "t" + i + "窗口").start();
        }
    }
}
